File: 02_EstructurasControl/Repetitivas/For/Python/Ejemplo02.py
# Pedir al usuario una cadena de texto para invertirla
cadena = input("Ingrese una cadena de texto para invertirla: ")
cadena_invertida = ""

# Recorrer la cadena de atrás hacia adelante para invertirla
for i in range(len(cadena) - 1, -1, -1):  # range descendente: desde len-1 hasta 0
    cadena_invertida += cadena[i]

print("La cadena invertida es: " + cadena_invertida)

File: 02_EstructurasControl/Repetitivas/For/C#/Ejemplo-02/Ejemplo02.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _2.For
{
    internal class Program
    {
        static void Main(string[] args)
        {
            //Recorre la cadena de atrás hacia adelante para invertirla.
            Console.WriteLine("Ingrese una cadena de texto para invertirla:");
            string cadena = Console.ReadLine();
            string cadena_invertida = "";
            for (int i = cadena.Length - 1; i >= 0; i--)
            {
                cadena_invertida += cadena[i];
            }
            Console.WriteLine("La cadena invertida es: " + cadena_invertida);
            Console.ReadKey();

        }
    }
}
